EIPS/eip-2256.md

```diff
@@ -0,0 +1,183 @@
+---
+eip: 2256
+title: wallet_getOwnedAssets JSON-RPC Method
+author: Loredana Cirstea (@loredanacirstea)
+discussions-to: https://ethereum-magicians.org/t/eip-2256-add-wallet-getownedassets-json-rpc-method/3600
+status: Draft
+type: Standards Track
+category: ERC
```

Comment:
I think this belongs under Interface where the other RPCs are.

Reply:
done.

Reply:
EIPS/eip-2256.md